### DISCUSSION TITLE
Jupyter Lab

### DISCUSSION THREAD

**Discussion by RicardoBatanero** [dat-linux/community#31] [Issues]
Hi, 
I´ve downloaded Jupyter Lab and I can launch it, but when I want to import packages like pandas,..., always appear error module not found. I downloaded in python 3 pip and installed them, but still is giving me same problem. Where can be the problem? how is supposed we should manage to download the different apps?
Thanks for your help.

**Comment by dat-linux** [comment]
Hi. Only packages needed to run Jupyter are installed by default. Any packages not installed in your env't need to be installed as you need them. You can do this from a Jupyter cell:
`!pip install pandas`
Or from the IPython console.

**Reply by RicardoBatanero** [reply]
Thx, that way I managed to download the packages
